test/test_minutes.py

```diff
@@ -0,0 +1,13 @@
+from minutes import Minutes
+import test.config as c
+
+
+def test_train():
+    minutes = Minutes(ms_per_observation=100)
```

Comment:
It might be good to go through and test model training with all of the models declared in `Minutes.models`.